elf/module.go

```diff
@@ -457,6 +458,10 @@ func (b *Module) closeMaps() error {
 	return nil
 }
 
+type CloseOptions struct {
```

Comment:
golint it is. There's more stuff it complains about that I'll do in a separate PR. Thanks!